addons/danger/functions/fnc_leaderStaticPack.sqf

```diff
@@ -0,0 +1,123 @@
+#include "script_component.hpp"
+/*
+ * Author: nkenny
+ * Pack static weapon
+ *
+ * Arguments:
+ * 0: Units <ARRAY>, <OBJECT>, <GROUP>
+ * 1: Range check <NUMBER>
+ * 2: Weapon to pack <ARRAY>
+ *
+ * Return Value:
+ * units in array
+ *
+ * Example:
+ * [units bob] call lambs_danger_fnc_leaderStaticPack;
+ *
+ * Public: No
+*/
+
+params ["_units", ["_guns", []]];
+
+// sort units
+if (_units isEqualType objNull) then { _units = [_units] call FUNC(findReadyUnits); };
+if (_units isEqualType grpNull) then { _units = [leader _units] call FUNC(findReadyUnits); };
+
+// get weapons list
+if (_guns isEqualTo []) then {
+    _guns = (group (_units select 0)) getVariable [QGVAR(staticWeaponList), []];
+};
+
+// check if weapon is unmanned
+_guns = _guns select {!(crew _x isEqualTo [])};
+if (_guns isEqualTo []) exitWith { _units };
+
+// get gunner
+private _weapon = _guns deleteAt 0;
+private _gunner = gunner _weapon;
+
+// check for nearest unit without backpack
+_units = _units select { isNull objectParent _x && { (backpack _x) isEqualTo ""} };
+
+// get nearest assistant ~ should probably write a function for this! - nkenny
+if (_units isEqualTo []) exitWith { _units };
+_units = _units apply { [ _x distance2D _gunner, _x ] };
+_units sort true;
+_units = _units apply { _x select 1 };
+
+// get assistant
+private _assistant = _units deleteAt 0;
```

Comment:
you now dont remove the gunner from the array of units

Reply:
L40 - sorts all units by bacpacks and objectParents.  The gunner is in a vehicle and is sorted out.